app/pktgen-stats.c

```diff
@@ -479,8 +479,9 @@ pktgen_page_stats(void)
  * SEE ALSO:
  */
 void
-pktgen_process_stats(struct rte_timer *tim __rte_unused, void *arg __rte_unused)
+pktgen_process_stats(struct rte_timer *tim __rte_unused, void *arg)
 {
+    uint64_t elapsed_ns = (uint64_t)arg;
```

Comment:
These changes look pretty good and it looks like you have fixed the rate problems, but I do have a few questions.

The elapsed_ns is stated here as a uint64_t and then converted to a double, can this be declared as a double and then the cast is not required later?

Reply:
This function is called from the RTE timer code and it requires the two parameters.